pytorch_lightning/core/module.py

```diff
@@ -531,6 +531,14 @@ def __check_allowed(v: Any, name: str, value: Any) -> None:
     def __to_tensor(self, value: numbers.Number) -> torch.Tensor:
         return torch.tensor(value, device=self.device)
 
+    @staticmethod
+    def __check_numel_1(value: torch.Tensor, name: str) -> None:
+        if not torch.numel(value) == 1:
```

Comment:
LGTM assuming extra dims get squeezed out?

Reply:
They don't. But everything is working as expected in some tests I run with multi-dim single-element tensors